### ISSUE TITLE
Add document language to HTML element

### ISSUE THREAD

**Issue by Roenok** [girldevelopit/gdi-new-site#125]
Per WCAG 2.0, the W3C's accessibility guidelines, HTML pages should ID their document language:

3.1.1 Language of Page: The default human language of each Web page can be programmatically determined. (Level A)

This makes it easier for assistive tech to pick the right captions, scripts, screenreader voice, etc. Instead of ```<html>```, the header should be ```<html lang="en-US">```.

For more, http://www.w3.org/TR/WCAG20-TECHS/H57.html

**Comment by coreyd303** [comment]
Should be able to just do this in application.html.erb, will do this and submit a PR

**Comment by coreyd303** [comment]
PR submitted

**Comment by juliaelman** [comment]
Tagging #127

**Comment by kstack7** [comment]
Now PR #167

**Comment by juliaelman** [comment]
Fixed with #167!
